path_2024.py

```diff
@@ -71,17 +71,17 @@ def main() -> int:
         print("Error: waypoints_to_commands")
         return -1
 
-    result, takeoff_landing_commands = (
-        add_takeoff_and_landing_command.add_takeoff_and_landing_command(
-            waypoint_commands,
-            ALTITUDE,
-        )
+    # get the last waypoint for loitering
+    loiter_coordinate = waypoints_list[-1]
```

Comment:
Nit: Rather than getting the end of the list and then passing them into function why not get the end of the list in the module and get the latitude and longitude from there.